antora-ui-camel/src/css/vars.css

```diff
@@ -18,7 +19,10 @@
   --color-black: #000;
   --color-asf-dark-blue: #303284;
   --color-asf-moderate-blue: #585ac2;
+  --color-bistre: #41250c;
   --color-camel-orange: #ed8225;
+  --color-camel-vivid-orange: #e97826;
+  --color-dark-gray: #767676;
```

Comment:
Could we use one of the existing gray shades we have or keep all the grey variables together and with consistent naming. This is 46% gray could we use 50% or 30% instead?

Reply:
I will try to make use of the 50% gray and make changes.

Reply:
46% gray provides better and more sophisticated contrast than the current 50% gray.